src/Resources/config/listeners.yml

```diff
@@ -0,0 +1,12 @@
+services:
+    terminal42.autoregistration.hook_listener.registration:
+        class: Terminal42\ContaoAutoRegistrationBundle\HookListener\Registration
+        arguments:
+            - '@contao.security.frontend_user_provider'
+            - '@security.token_storage'
+            - '@session'
+            - '@database_connection'
+            - '@monolog.logger.contao'
+        tags:
+            - { name: 'contao.hook', hook: 'createNewUser', method: 'processRegistration' }
```

Comment:
Can we rename the methods to match the host names? If you rename `processRegistration` to `onCreateNewUser` then we don't need to set method in the tag.